js/generate.js

```diff
@@ -140,18 +158,10 @@ function fillTables() {
 		var implTd = document.createElement("td");
 		var docTd = document.createElement("td");
 		var tsTd = document.createElement("td");
-		var xhr = new XMLHttpRequest();
-		xhr.open("GET", "data/" + spec + ".json");
-                counterReq++;
-		xhr.onload = function(x, s, el1, el2, el3, el4, el5, el6, el7, el8) {
-		    return function() {
-                        counterRes++;
+                
+		function trClosure(s, el1, el2, el3, el4, el5, el6, el7, el8) {
```

Comment:
:heart: ... but :open_mouth:!? If you are going ES6 here, then maybe use `...elems` as a rest parameter. Then you can access them in array order. 

```JS
function trClosure(s, ...elems) {
}
```